config/environments/production.rb

```diff
@@ -43,7 +43,7 @@
 
   # Use the lowest log level to ensure availability of diagnostic information
   # when problems arise.
-  config.log_level = :debug
+  config.log_level = :error
```

Comment:
I'd probably `info` or `warn` here. wouldn't error clobber the others?

Reply:
we might even want to make that variable if all of a sudden we wanted to drop prod to debug mode without a full deployment.